Sources/SwifterSwift/Dispatch/DispatchQueueExtensions.swift

```diff
@@ -52,7 +45,8 @@ public extension DispatchQueue {
     func asyncAfter(delay: Double,
                     qos: DispatchQoS = .unspecified,
                     flags: DispatchWorkItemFlags = [],
-                    execute work: @escaping () -> Void) {
+                    execute work: @escaping () -> Void)
+    {
```

Comment:
Is this a bug, maybe line width maybe conflicting with the same-line bracet rule ... ?

Reply:
I suspect it always uses a newline when parameter are wrapped. Probably to give a cleaner indent after the bracket. Not sure, though.

Reply:
Maybe we can disable `wrapMultilineStatementBraces`